plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/dialogs/SuggestionInformationControl.java

```
@@ -151,35 +151,51 @@ private void createTreeControl(JDBCTable<?, ?> input) {
             .getActivePart()
             .getSite());
         DBNDatabaseNode node = DBWorkbench.getPlatform().getNavigatorModel().findNode(input);
-        Collection<DBNNode> columnNodes;
-        try {
-            columnNodes = getColumnNodes(node);
-            if (columnNodes.isEmpty()) {
-                return;
+        new AbstractJob("Populating table tip") {
+            @Override
+            protected IStatus run(DBRProgressMonitor monitor) {
+                monitor.beginTask("load table columns", 1);
+                try {
+                    Collection<DBNNode> columnNodes;
+                    columnNodes = getColumnNodes(monitor, node);
+                    if (columnNodes.isEmpty()) {
+                        return Status.OK_STATUS;
+                    }
+                    UIUtils.asyncExec(() -> {
+                        ItemListControl itemListControl = new ItemListControl(
+                            tableComposite,
+                            SWT.NONE,
+                            subSite,
+                            node,
+                            null
+                        );
+                        itemListControl.setLayoutData(gridData);
+                        itemListControl.appendListData(columnNodes);
+                        tableComposite.layout(true, true);
+                    });
+                } catch (DBException e) {
+                    return Status.CANCEL_STATUS;
```

Please don't suppress errors. Use `GeneralUtils#makeExceptionStatus`

If you believe that we should not show the thrown exception, then at least log it and write what went wrong.